content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -130,7 +130,7 @@ <h3>Deploy an app</h3>
         <div class="row">
             <div class="col-md-12">
                 <h3>View the app</h3>
-                <p>Pods that are running inside Kubernetes are running on a private, isolated network.
+                <p><a href="/docs/concepts/workloads/pods/">Pods</a> that are running inside Kubernetes are running on a private, isolated network.
```

Comment:
This helps; I would also like to tell the reader that right now we're not explaining what Pods are in any detail (so they know it's OK to continue, confident they'll learn later).

Reply:
@sftim,
Thanks for the review, updated the text.